tests/cmds/test_cases.py

```diff
@@ -0,0 +1,74 @@
+from code42cli.main import cli
+
+
+def test_create_case_calls_create_with_expected_params(runner, cli_state):
+    runner.invoke(
```

Comment:
For the `create_` and the `update_` tests, can you use all the options so that we can have the tests verify that each of the options gets passed into py42?

Reply:
Done.